dali/CMakeLists.txt

```diff
@@ -226,3 +209,27 @@ if (BUILD_PYTHON)
   # Get all python srcs
   add_subdirectory(python)
 endif()
+
+
+################################################
+# Gather DALI headers for whl
+################################################
+
+# Copy all headers from DALI_INST_HDRS list to DALI_WHEEL_DIR using install command
+# with `-D` option, that recursively creates missing directories in destination path
+add_custom_target(install_headers ALL
+    DEPENDS ${dali_lib}
+)
+
+# Process the DALI_INST_HDRS list
+foreach(INSTALL_HEADER ${DALI_INST_HDRS})
+  add_custom_command(
+    TARGET install_headers
+    COMMAND install -D "${CMAKE_SOURCE_DIR}/${INSTALL_HEADER}" "${PROJECT_BINARY_DIR}/${DALI_INCLUDE_DIR}/${INSTALL_HEADER}")
+endforeach(INSTALL_HEADER)
+
+# Copy proper `include` dir
+add_custom_command(
+  TARGET install_headers
+  COMMAND cp -r "${CMAKE_SOURCE_DIR}/include/." "${PROJECT_BINARY_DIR}/${DALI_INCLUDE_DIR}"
```

Comment:
```suggestion
  COMMAND cp -r "${CMAKE_SOURCE_DIR}/include" "${PROJECT_BINARY_DIR}/${DALI_INCLUDE_DIR}"
```

Reply:
This would copy the `${CMAKE_SOURCE_DIR}/include` into `${PROJECT_BINARY_DIR}/${DALI_INCLUDE_DIR}`, and we would end up with `include/include/something`.